script/download_rust_deps.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# Copyright 2017 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""Script to download rust_deps."""
+
+import os
+import re
+import subprocess
+import sys
+import urllib2
+
+import deps
+
+RUST_DEPS_URL = "https://s3-us-west-2.amazonaws.com/rust-pkg-brave-core"
+RUST_DEPS_VERSION = "0.1.0"
```

Comment:
we should have this in either python (as an import for both scripts) or gn, but not both

Reply:
👍